### PR TITLE
Wiremock

### PR DESCRIPTION
**Pull request by RAJKUMAR733**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -34,6 +34,7 @@ dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.cloud:spring-cloud-starter-openfeign'
 	implementation 'org.springframework.kafka:spring-kafka'
+	implementation 'org.projectlombok:lombok:1.18.22'
```
Comment (samratshekhar) [inline build.gradle:37]: lombok is already added. Why is this required?

File: src/test/java/com/example/kafka/kafkaservice/integration/LookUpServiceClientTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        wireMockServer.start();
+        WireMock.configureFor(wireMockServer.port());
+    }
+
+    @AfterEach
+    void stopServer() {
+        wireMockServer.stop();
+    }
+    @Test
+    void executeRequest() throws IOException {
```
Comment (samratshekhar) [inline src/test/java/com/example/kafka/kafkaservice/integration/LookUpServiceClientTest.java:48]: Rename to something meaningful

File: src/test/java/com/example/kafka/kafkaservice/integration/LookUpServiceClientTest.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        LookUpSyncRequestDto lookUpSyncRequestDto=new
+                LookUpSyncRequestDto("test12","test","test");
+        WireMock wireMock=new WireMock(wireMockServer.port());
+        wireMock.register(any(urlPathMatching("/api/lookup"))
+                .withHeader("customer-id",equalTo("1234567890"))
+                .withHeader("ip-address",equalTo("XXX.XXX.XXX.XXX"))
+                .withRequestBody(equalToJson(new Gson().toJson(lookUpSyncRequestDto)))
+                .willReturn(aResponse().withHeader("Content-Type", "application/json").withBody(new Gson().toJson(accountDetailsDto))));
+        int port= wireMockServer.port();
+        String responseUrl=wireMockServer.baseUrl();
```
Comment (samratshekhar) [inline src/test/java/com/example/kafka/kafkaservice/integration/LookUpServiceClientTest.java:61]: where is port and url used?

File: build.gradle (hunk 1 above)
Comment (RAJKUMAR733) [reply build.gradle:37]: when i pushed, it got added. I checked with git status but it didnt show  in commit logs.

**Review by samratshekhar — APPROVED** [review]
